### PR TITLE
Update test dependencies

### PR DESCRIPTION
**Pull request by tmorrell**
:heart: Thank you for your contribution!

### Description

Update versions so tests pass

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Review by fenekku — APPROVED** [review]
Looks good to me!

I will narrate here what I did (à-la UX testing session) to make explicit the real flow I went through (we can talk about it better during our PR triage meeting). So:

I saw the PR via email because I was assigned as reviewer. I went to the Triage board and saw that it was in "To Review", so I moved it to "In Review". Then I started reviewing it. It's all good for me, so I approved it. I left it in the "In Review" column, since @kpsherva was also assigned as a Reviewer. I didn't assign myself to it either since I was already marked as a reviewer. 

My involvement is done, and I am expecting Karolina to review and eventually place it in "Approved" after any discussion. With Tom being the main maintainer of the package (in addition to being submitter which is a common scenario), I expect he will determine the right release schedule with respect to the current goings on.

**Comment by tmorrell** [comment]
I'm also going to narrate a bit as PR coordinator. I know [kpsherva](https://github.com/kpsherva) is away this week, so I'm going to swap CERN reviewers and add nicola. I'm leaving this in the "in review/discussion" column, although you could argue we're just waiting for an additional review.

I'm not sure what mechanism works best for managing reviews - should we expect reviewers to look at the board or should the coordinator directly assign reviewers? I'm trying the latter for the next week.

File: .github/workflows/tests.yml
Comment (ntarocco) [inline .github/workflows/tests.yml:31]: Minor: for the line below this one, in general we are removing the tests of `min`, keeping only the test for the latest. See for example [here](https://github.com/inveniosoftware/invenio-communities/blob/master/.github/workflows/tests.yml#L33).

**Review by ntarocco — APPROVED** [review]
LGTM!
I have added a minor comment, not really related to the changes in this PR, just because I noticed it. I have also noticed that, at some point, we might want to apply the same module upgrades/changes that @utnapischtim has done in many modules, as for example [here](https://github.com/inveniosoftware/invenio-app/pull/85). Something to do in the future :)

**Comment by ntarocco** [comment]
I moved the PR to the `approved` column (should I have done that? :))

File: .github/workflows/tests.yml
Comment (tmorrell) [reply .github/workflows/tests.yml:31]: Awesome-that will make maintenance easier!

**Pull request merged**
